examples/simple/src/index.tsx

```diff
@@ -14,13 +14,18 @@ import Layout from './Layout';
 import posts from './posts';
 import users from './users';
 import tags from './tags';
+import { queryClient } from './queryClient';
 
-render(
+const container = document.getElementById('root');
+const root = createRoot(container);
+
+root.render(
     <React.StrictMode>
         <Admin
             authProvider={authProvider}
             dataProvider={dataProvider}
             i18nProvider={i18nProvider}
+            queryClient={queryClient}
```

Comment:
As it's the standard query client, I don't understand why you need to specify it.